Doc/library/codecs.rst

```diff
@@ -1126,7 +1126,8 @@ particular, the following variants typically exist:
 +-----------------+--------------------------------+--------------------------------+
 | cp875           |                                | Greek                          |
 +-----------------+--------------------------------+--------------------------------+
-| cp932           | 932, ms932, mskanji, ms-kanji  | Japanese                       |
+| cp932           | 932, ms932, mskanji, ms-kanji, | Japanese                       |
+|                 | windows_31j                    |                                |
```

Comment:
In this table, hiphen is used more than underscore. e.g. "iso-2022-jp-2004", "windows-1251".
So "windows-31j" looks more natural to me.

Reply:
Thank you for your advice.
Fixed the description of Doc/library/codecs.rst.

Reply:
The modifications are as follows:
https://github.com/python/cpython/compare/e2371988360b156d2f755e855df06b18cf66f46b..566d1e0403009baa66e47d387c98b073c57f2f0d